### PR TITLE
Fix centering of main content

### LINKED ISSUE #58: Main content is not centered
The layout is currently not very nice (tested both on Firefox and Chrome).
For example:
- I would expect the table to be centered in the page
- I would expect the "Previous" and "Next" buttons to be aligned with the right border of the table, (similarly as the pages  1, 2, etc. buttons are aligned to the left border)

Screenshot from 2022-09-27 08-33-41

### PR DESCRIPTION
**Pull request by omichel**
Fixes #58 (was pushed on https://testing.webots.cloud for testing).

Note: this PR introduces empty lines on the last page of the tables to ensure the height of the page is always the same.

### CODE REVIEW

**Comment by BenjaminDeleze** [comment]
Honestly I am not super fan of the new layout.

I prefer the one where everything was aligned left.

I think that the blanks (especially the right margin) feel too big.

File: httpdocs/css/webots-cloud.css [hunk 1]
```diff
@@ -34,7 +34,12 @@ body {
 }
 
 div.table-container {
-  min-height: 550px;
+  display: grid;
+  justify-content: center;
```
Comment (omichel) [inline httpdocs/css/webots-cloud.css]: Alternatively, removing this line `justifiy-content: center;` will make the table spread over the full width of the page. Would that look any better?

**Comment by BenjaminDeleze** [comment]
center
large
left


I think my favorite is center and spread but I am not totally convinced

**Comment by omichel** [comment]
I changed to have the table spread over the full width of the page, which also looks better to me. I also fixed the width of all the columns except the one of the title.

**Comment by stefaniapedrazzi** [comment]
Can this be tested on `testing.webots.cloud`?

**Comment by BenjaminDeleze** [comment]
yes, I think you can see it on testing.webots.cloud/animation

**Comment by stefaniapedrazzi** [comment]
It is fine to have the table spread over the full width. 
But the position of the "Add a new..." button is now strange as it is not centered or aligned with anything.

Then, it looks bad in my opinion how the table is now displaying the empty rows.
It was much better before where no rows where shown.

Tested on Firefox.
testing.webots.cloud:
Screenshot from 2022-09-29 08-29-56


webots.cloud:
image

**Comment by omichel** [comment]
Empty rows should only be visible on the last page, which will soon become not easily visible. I added these empty rows to justify the fact that the buttons remains at the bottom position, otherwise, it looks strange to have an empty white space and the buttons below this empty space. I will try to remove these empty lines while keeping the equivalent white space to see if that looks good, but I am not sure.

**Comment by omichel** [comment]
>  But the position of the "Add a new..." button is now strange as it is not centered or aligned with anything.

I agree, I will fix that and align it to the left hand side.

**Comment by omichel** [comment]
I fixed all the problems reported so far. In particular, I removed the empty rows on the last table. Can you review it again and approve or ask for changes?

**Comment by stefaniapedrazzi** [comment]
Thank you! I confirm all the reported problems are now fixed.

With respect to the previous version, on the server tab there is much more empty space before the table. Is this wanted?
Because it seems better in the previous version.

Just for note I also noticed that the tables rows height is now slightly bigger than before, but it is not a problem.

**Comment by BenjaminDeleze** [comment]
I would prefer if there is no grey line after the last `simulation`/`scene`/...

It is nice to have it between two rows but not at the end I think

**Comment by omichel** [comment]
> With respect to the previous version, on the server tab there is much more empty space before the table. Is this wanted?
Because it seems better in the previous version.

Yes, this due to the fact that there is no search tool for servers now. I wanted to reserve this space for when we will add a search tool there.

**Comment by omichel** [comment]
I fixed the last problems reported. Can you review it again and approve or ask for changes?

**Review by BenjaminDeleze — APPROVED** [review]
Seems good for me

**Pull request merged**